rosa_getting_started/rosa-quickstart-guide-ui.adoc

```diff
@@ -118,6 +118,10 @@ include::modules/rosa-getting-started-grant-user-access.adoc[leveloffset=+2]
 [discrete]
 include::modules/rosa-getting-started-grant-admin-privileges.adoc[leveloffset=+2]
 
+.Additional resources
+
+* For more information about `cluster-admin` privileges, see xref:../rosa_architecture/rosa_policy_service_definition/rosa-service-definition.adoc#rosa-sdpolicy-cluster-admin-role_rosa-service-definition[Cluster administration role].
+* For more information about `dedicated-admin` privileges, see xref:../rosa_architecture/rosa_policy_service_definition/rosa-service-definition.html#rosa-sdpolicy-customer-admin-user_rosa-service-definition[Cluster administrator user].
```

Comment:
See OpenShift docs guidelines for the [Additional resources sections](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections).
```suggestion
[role="_additional-resources"]
.Additional resources

* xref:../rosa_architecture/rosa_policy_service_definition/rosa-service-definition.adoc#rosa-sdpolicy-cluster-admin-role_rosa-service-definition[Cluster administration role].
* xref:../rosa_architecture/rosa_policy_service_definition/rosa-service-definition.html#rosa-sdpolicy-customer-admin-user_rosa-service-definition[Cluster administrator user].
```

Reply:
@bhardesty We are not taking this suggestion?

Reply:
@mburke5678 Yes! I missed it somehow...I've fixed it now.